src/slash-bedrock/share/brl-fetch/distros/chimera

```diff
@@ -0,0 +1,77 @@
+#!/bedrock/libexec/busybox sh
+#
+# Chimera Linux bootstrap support
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted;
+# According to the terms of the BSD Zero Clause License.
+# 
+# Copyright (c) 2024 Anthony Slab
+#
+
+# shellcheck source=src/slash-bedrock/libexec/brl-fetch
+. /bedrock/share/common-code
+trap 'fetch_abort "Unexpected error occurred."' EXIT
+
+check_supported() {
+	false
+}
+
+speed_test_url() {
+	echo "current/main/${distro_arch}/APKINDEX.tar.gz"
+}
+
+list_mirrors() {
+	# pretty sure there's only the official one :p
+	echo "https://repo.chimera-linux.org"
+}
+
+brl_arch_to_distro() {
+	case "${1}" in
+	"aarch64") echo "aarch64" ;;
+	"riscv64") echo "riscv64" ;;
+	"ppc64")	echo "ppc64" ;;
+	"ppc64le") echo "ppc64le" ;;
+	"x86_64")  echo "x86_64" ;;
+	*) abort "brl does not know how to translate arch \"${1}\" to ${distro} format" ;;
+	esac
+}
+
+list_architectures() {
+	cat <<eof
+aarch64
+riscv64
+ppc64
+ppc64le
+x86_64
+eof
+}
+
+default_release() {
+	echo "current"
+}
+
+list_releases() {
+	echo "current"
+}
+
+fetch() {
+	step "Downloading bootstrap software"
+	url="$(find_link "${target_mirror}/apk/" "^apk-${target_arch}-.*.static")"
+	apk="$(basename ${url})"
+	download "${url}" "${bootstrap_dir}/${apk}"
+
+	download "${target_mirror}/apk/sha256sums.txt" "${bootstrap_dir}/shasums"
+	[ "$(sha256sum ${bootstrap_dir}/${apk} | tr -d' ' -f1)" = "$(grep ${apk} ${bootstrap_dir}/shasums | tr -d' ' -f1)" ] \
+		|| { echo "ERROR: checksums did not match"; false; }
+
+	step "Preparing bootstrap software"
+	chmod a+rx "${bootstrap_dir}/${apk}"
+	mkdir -p "${target_dir}/etc/apk/"
+
+	step "Running bootstrap software"
+	setup_chroot "${bootstrap_dir}"
+	setup_ssl "${bootstrap_dir}"
+	LC_ALL=C chroot "${bootstrap_dir}" "/${apk}" -X "${target_mirror}/${target_release}/main" -U \
+		--allow-untrusted --root "/target-root" --initdb add chimerautils base-minimal
```

Comment:
Drop `chimerautils`, `base-minimal` is a smaller subset of [`base-full`](https://github.com/chimera-linux/cports/blob/master/main/base-full/template.py) which already depends on [`base-bootstrap`](https://github.com/chimera-linux/cports/blob/master/main/base-bootstrap/template.py) pulling that in implicitly